backend/app/views/spree/layouts/admin.html.erb

```diff
@@ -19,7 +19,7 @@
                 <%= yield :tabs %>
               </div>
             <% end %>
-            <div class="<%= if content_for?(:sidebar) then 'twelve' else 'sixteen' end %> columns">
+            <div id="content-without-nav" class="<%= if content_for?(:sidebar) then 'twelve' else 'sixteen' end %> columns">
```

Comment:
Totally. I have completely reworked this for future layouts because I ran into the same problem: https://github.com/solidusio/solidus/compare/master...graygilmore:content_layout#diff-b1a47ee5eb885f91f6fe5ff9b2be5211R13

Since `#content-without-nav` is only being used for a JS hook maybe just something like `class="js-content-below-tabs`? Either way it's going to be ugly but I'd rather avoid something like `content-without-nav` as it implies that there is no nav on the page (if you were viewing it out of context, like in the `new.js.erb` file).